### PR TITLE
Changes to support gather QA

### PR DESCRIPTION
**Pull request by tqorange**
Also fixed the NewEventType function parameter names.

### CODE REVIEW

File: ftrace/ftrace.go [hunk 1]
```diff
@@ -65,8 +65,8 @@ func (f *Ftrace) init() error {
 	return nil
 }
 
-func (f *Ftrace) NewEventType(path string, name string) (*EventType, error) {
-	etype, err := newEventType(f.fp, path, name)
+func (f *Ftrace) NewEventType(name string, path string) (*EventType, error) {
```
Comment (nkirsch) [inline ftrace/ftrace.go]: I presume there are changes to callers as well? Who wrote this code, anyway?! 👍

**Comment by nkirsch** [comment]
Looks good to me!

File: ftrace/ftrace.go (hunk 1 above)
Comment (tqorange) [reply ftrace/ftrace.go]: Callers should be ok, only the names were changed, not the order. I guess if the callers also used the `incorrect` names, then, we'd want to change them.

Comment (nkirsch) [reply ftrace/ftrace.go]: 🥇

Comment (tqorange) [reply ftrace/ftrace.go]: Existing callers look fine however.

**Pull request merged**